assets/js/Ioda/pages/dashboard/Dashboard.js

```diff
@@ -240,313 +238,293 @@ class Dashboard extends Component {
-          : this.state.activeTabType === region.type
-          ? (topoItemIndex = this.state.topoData.features.findIndex(
-              (topoItem) => topoItem.properties.name === outage.entity.name
+        activeTabType === country.type
+            ? (topoItemIndex = topoData.features.findIndex(
+                (topoItem) => topoItem.properties.usercode === outage.entity.code
             ))
-          : null;
+            : activeTabType === region.type
+                ? (topoItemIndex = topoData.features.findIndex(
+                    (topoItem) => topoItem.properties.name === outage.entity.name
+                ))
+                : null;
 
         if (topoItemIndex > 0) {
-          let item = topoData.features[topoItemIndex];
+          let item = _topoData.features[topoItemIndex];
           item.properties.score = outage.scores.overall;
-          topoData.features[topoItemIndex] = item;
+          _topoData.features[topoItemIndex] = item;
 
           // Used to determine coloring on map objects
           scores.push(outage.scores.overall);
           scores.sort((a, b) => a - b);
         }
       });
-      this.setState({ topoScores: scores });
+      setTopoScores(scores);
     }
-  };
+  }
+
   // Make API call to retrieve topographic data
-  getDataTopo = (entityType) => {
+  function getDataTopo(entityType) {
+    console.log('305')
     let topologyObjectName =
-      entityType == "country"
-        ? "ne_10m_admin_0.countries"
-        : "ne_10m_admin_1.regions";
-    if (this.state.mounted) {
+        entityType === "country"
+            ? "ne_10m_admin_0.countries"
+            : "ne_10m_admin_1.regions";
+    if (mounted) {
+      console.log('mounted')
       getTopoAction(entityType)
-        .then((data) =>
-          topojson.feature(
-            data[entityType].topology,
-            data[entityType].topology.objects[topologyObjectName]
+          .then((data) =>
+              topojson.feature(
+                  data[entityType].topology,
+                  data[entityType].topology.objects[topologyObjectName]
+              )
           )
-        )
-        .then((data) =>
-          this.setState(
-            {
-              topoData: data,
-            },
-            this.getMapScores
-          )
-        );
+          .then((data) => {
+            console.log(data)
+            setTopoData(data)
+          })
     }
-  };
-  // function to manage when a user clicks a country in the map
-  handleEntityShapeClick = (entity) => {
-    const { navigate } = this.props;
+    console.log('328')
+  }
 
+  // function to manage when a user clicks a country in the map
+  function handleEntityShapeClick(entity) {
     // Use usercode for country, id for other types
     const entityCode =
-      this.state.activeTabType === country.type
-        ? entity.properties.usercode
-        : entity.properties.id;
-    let path = `/${this.state.activeTabType}/${entityCode}`;
+        activeTabType === country.type
```

Comment:
Your IDE formatter settings appear to be changing the whitespace of this code. Can you make sure your formatter is set to 2 spaces for tab, instead of 4?